packages/docs/link-docs/stories/link.react.meta.ts

```diff
@@ -0,0 +1,35 @@
+import type { Meta } from '@storybook/react-vite';
+import { Link } from '@nl-design-system-candidate/link-react/css';
+
+const meta = {
+  argTypes: {
+    children: { table: { category: 'API' }, type: 'string' },
+    href: { type: { required: true }, table: { category: 'API' } },
```

Comment:
Sommige properties werken niet optimaal in Storybook, die zouden fijner werken als string of combobox.

`href` is een regressie, die was een string en is nu JSON.

<img width="802" height="499" alt="Image" src="https://github.com/user-attachments/assets/a0827154-582e-432a-a8a6-470e1409b280" />

`current` werkte al niet ideaal